internal/domain/customer/model.go

```diff
@@ -142,3 +145,44 @@ func ValidateAddress(c *Customer) error {
 	}
 	return nil
 }
+
+func (c *Customer) ToPdfgenRecipientInfo() *pdfgen.RecipientInfo {
```

Comment:
on 2nd thoughts I should not be doing this, could lead to circular imports in the future